doc/radosgw/frontends.rst

```diff
@@ -130,10 +130,13 @@ Options
 :Default: ``EMPTY``
 
 
-The following is an example of the ``/etc/ceph/ceph.conf`` file with some of these options set:
+The following is an example of the ``/etc/ceph/ceph.conf`` file with some of these options set: ::
  
  [client.rgw.gateway-node1]
- rgw frontends = civetweb request_timeout_ms=30000 error_log_file=/var/log/radosgw/civetweb.error.log access_log_file=/var/log/radosgw/civetweb.access.log
+ rgw_frontends = civetweb 
+ request_timeout_ms = 30000 
+ error_log_file = /var/log/radosgw/civetweb.error.log 
+ access_log_file = /var/log/radosgw/civetweb.access.log
```

Comment:
sorry, i didn't catch this before it merged. all of these values need to be on the same line as part of the `rgw_frontends` value. this new config example won't produce the same result

Reply:
@cbodley sorry for this! will send a fix late tomorrow.

Reply:
@tchaikov see https://github.com/ceph/ceph/pull/27054

Reply:
@tchaikov @cbodley My apologies for causing this error. If there's anything I can do fix, please let me know.

Reply:
@falcon78921 no worries. the upside is that @cbodley spotted this problem and we do have a fix =)